sys/include/xtimer.h

```diff
@@ -39,8 +41,6 @@
  * @author  Kaspar Schleiser <kaspar@schleiser.de>
  * @author  Joakim Nohlgård <joakim.nohlgard@eistec.se>
  */
-#ifndef XTIMER_H
-#define XTIMER_H
```

Comment:
```C
#define XTIMER_H /* This macro is used to check whether internal headers have been included via the public header */
```

The define here needs to stay (see comment in the sugesstion for the reason). But the `#ifndef`...`#endif` is no longer needed due to `#pragma once`